utils/metadata.ts

```diff
@@ -0,0 +1,13 @@
+// Grabs the demensions and other info of a transformed image
+export const fetchMetadata = async (metadataUrl: string) => {
```

Comment:
You can type the response too for more type safety

Reply:
I pulled a @SferaDev on you and already had this coming! Check it out.